tests/code/test_minor.py

```diff
@@ -132,26 +134,345 @@ def test_getCommunityEngagementByTerm():
+        User.update(minorInterest = 1).where(User.username == 'khatts').execute()
+        minorProgress = getMinorInterest()
+        assert minorProgress == []
+       
+       # Add a student will be returned in the list
+        User.update(minorInterest = 1).where(User.username == 'partont').execute()
+        oneStudentInterested = getMinorInterest()
+        assert len(oneStudentInterested) == 1
+        oneStudentInterested[0]['username'] == 'partont'
+
+        transaction.rollback()
+
+@pytest.mark.integration
+def test_getMinorProgress():
+    with mainDB.atomic() as transaction: 
+        # Make sure the individual requirement table is empty. 
+        IndividualRequirement.delete().execute()
+        noMinorProgress = getMinorProgress()
+
+        assert noMinorProgress == []
+        
+        khattsSustainedEngagement = {"username": "khatts",
+                                     "program": 2,
+                                     "course": None,
+                                     "description": None,
+                                     "term": 3,
+                                     "requirement": 14,
+                                     "addedBy": "ramsayb2",
+                                     "addedOn": "",
+                                     }
+
+        IndividualRequirement.create(**khattsSustainedEngagement)
+        khattsProgress = getMinorProgress()
+        
+        assert khattsProgress[0]['engagementCount'] == 1
+        assert khattsProgress[0]['hasSummer'] == "Incomplete"
+
+        khattsSummerEngagement = {"username": "khatts",
+                                  "program": None,
+                                  "course": None, 
+                                  "description": "Summer engagement",
+                                  "term": 3,
+                                  "requirement": 16,
+                                  "addedBy": "ramsayb2",
+                                  "addedOn": ""
+                                 }
+        
+        IndividualRequirement.create(**khattsSummerEngagement)
+        khattsProgressWithSummer = getMinorProgress()
+
+        assert khattsProgressWithSummer[0]['engagementCount']== 1
+        assert khattsProgressWithSummer[0]['hasSummer'] == "Completed"
+        
 
-        assert actual_result == expected_result
         transaction.rollback()
+
+@pytest.mark.integration
+def test_saveSummerExperience():
+    with mainDB.atomic() as transaction: 
+        IndividualRequirement.delete().execute()
+
+        # Add summer Experience for a user 
+        partontSummerExperience = {"summerExperience": "Test Summer Experience for Tyler",
+                                   "selectedSummerTerm": "Summer 2021",
+                                   }
+        
+        saveSummerExperience('partont', partontSummerExperience, 'ramsayb2')
+
+        getPartontSummerExperience = IndividualRequirement.select()
```

Comment:
I have the same comments for this test as I do for the last one. Feel like we should get the IndividualRequirements for a particular person, not for everyone.